src/Nox.Lib/Presentation/Api/OData/ApiRouteEtagOperationFilter.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Nox.Presentation.Api.OData;
+
+public class ApiRouteEtagOperationFilter:  IOperationFilter
+{
+    private  static readonly  string[] _eTagRequiredHttpMethods = { HttpMethod.Delete.Method, HttpMethod.Patch.Method, HttpMethod.Put.Method };
```

Comment:
suggest to use a hashset, since is mainly used for search

Reply:
Desired changes have been applied